### DISCUSSION TITLE
Use asp-append-version="true" to stop caching

### DISCUSSION THREAD

**Discussion by robalexclark** [Megabit/Blazorise#2688] [Ideas]
I notice that you added the version no after the css and js to stop caching, but you can use asp-append-version="true" and you won't need to change it for each new version

More info here:
https://www.c-sharpcorner.com/blogs/aspappendversion-feature-in-asp-net-core

**Comment by stsrki** [comment]
It's a nice feature but unfortunately, it only works for Blazor Server. On WebAsemmbly we have index.html so `asp-append-version="true"` has no effect.
